src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01-browser.md

```diff
@@ -19,25 +23,108 @@ export default function () {
 - [browser.close()](#browser-close)
 - [browser.contexts()](#browser-contexts)
 - [browser.isConnected()](#browser-isconnected)
-- 🚧 [browser.newBrowserCDPSession()](#browser-newbrowsercdpsession)
 - [browser.newContext([options])](#browser-newcontext-options)
 - [browser.newPage([options])](#browser-newpage-options)
+- [browser.version()](#browser-version)
+- 🚧 [browser.newBrowserCDPSession()](#browser-newbrowsercdpsession)
 - 🚧 [browser.on()](#browser-on)
 - ❌ [browser.startTracing()](#browser-starttracing)
 - ❌ [browser.stopTracing()](#browser-stoptracing)
-- [browser.version()](#browser-version)
 
 
 ## browser.close()
 
+In case this browser is obtained using [browserType.launch([options])](03-browser-type.md#launch), closes the browser and all of its pages (if any were opened).
+
+In case this browser is connected to, clears all created contexts belonging to this browser and disconnects from the browser server.
+
+The [Browser](01-browser.md) object itself is considered to be disposed and cannot be used anymore.
+
 ## browser.contexts()
 
+Returns an array of all open browser contexts. In a newly created browser, this will return zero browser contexts.
+
+<!-- eslint-skip -->
+
+```javascript
+const browser = launcher.launch('chromium');
+console.log(browser.contexts().length); // prints `0`
+
+const context = browser.newContext();
+console.log(browser.contexts().length); // prints `1`
+```
+
 ## browser.isConnected()
 
+Indicates that the browser is connected.
```

Comment:
We should also mention the return value/type in the same format we use in the k6 JS API.

For example, take a look at [`check()`](https://k6.io/docs/javascript-api/k6/check).